### ISSUE TITLE
Duda learning rate

### ISSUE THREAD

**Issue by Regenschrim** [PUC-RecSys-Class/RecSysPUC-2020#27]
En la parte de ALS nos piden jugar modificando el learning_rate del algortimo, pero por lo que estuve buscando ninguna implemenctación (de pyreclab o implicit) tiene la opción de modificar el learning_rate, ¿Es un typo o si tiene la opción pero hay otro modo de modificarla?

Gracias!

**Comment by mcartagenah** [comment]
Efectivamente es un typo, no hay que variar ese parámetro en ALS ya que no aplica.

Saludos!

**Comment by Regenschrim** [comment]
Bacan, gracias!!
